compiler/natives/src/testing/example.go

```diff
@@ -48,11 +36,16 @@ func runExample(eg InternalExample) (ok bool) {
 		// Close pipe, restore stdout, get output.
 		w.Close()
 		os.Stdout = stdout
-		out := <-outC
+		out, e := ioutil.ReadFile(w.Name())
+		if e != nil {
+			fmt.Fprintf(os.Stderr, "testing: reading stdout file: %v\n", e)
+			os.Exit(1)
+		}
+		_ = os.Remove(w.Name())
```

Comment:
If there's any error during reading the example stdout file, `_ = os.Remove(w.Name())` line will never run.

Would the following be better?

```Go
out, readFileError := ioutil.ReadFile(w.Name())
os.Remove(w.Name())
if readFileError != nil {
	fmt.Fprintf(os.Stderr, "testing: reading stdout file: %v\n", readFileError)
	os.Exit(1)
}
```